LiteDB/Utils/Logging.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Diagnostics;
+using System.Text;
+
+namespace LiteDB
+{
+    public static class Logging
+    {
+        public static event Action<LogEventArgs> LogCallback;
```

Comment:
Have at least some simple mechanism to hook to logs and log them - to reveal what is wrong if it fails in production